zebrad/src/components/sync/end_of_support.rs

```diff
@@ -13,7 +13,7 @@ use zebra_chain::{
 use crate::application::release_version;
 
 /// The estimated height that this release will be published.
-pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_373_686;
+pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_413_000;
```

Comment:
Nitpick/Optional: It should take a couple days for the full sync test to finish.

```suggestion
pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_415_000;
```

Reply:
I was planning to finish the release today. The job was successful for the branch that fixed the issue, so I wasn't going to wait for `main`.